web_accounts_list.json

```diff
@@ -3154,7 +3165,18 @@
       "account_missing_code" : "200",
       "known_accounts" : ["test"],
       "category" : "social",
-      "valid" : true
+      "valid" : true
+    },
+    {
+      "name" : "Pokec",
+      "check_uri" : "https://pokec.azet.sk/{account}",
+      "account_existence_code" : "200",
+      "account_existence_string" :"nick",
```

Comment:
I am concerned that, while this works now, the website changing and having the term `nick` on its main page could occur. Can you find a better, more unique existence code please?

Reply:
> I am concerned that, while this works now, the website changing and having the term `nick` on its main page could occur. Can you find a better, more unique existence code please?

- `nick` can only be present if a profile is found (200) as it means "Nickname".
- I checked in Slovak Language and there is no word for nick so there is little chance it would show on the main page, but I found a better one:
-`idReportedUser`

Reply:
@WebBreacher All changes you required are: Done✅

Reply:
Understood about the term `nick`. I was concerned about the site putting some word containing the term `nick` into their page and false positives occuring. The way the `account_existence_string` works is that it matches that text in the response. So the word `beatnick` (I realize the site is not in English) on the error page would be seen as a positive account match since the term `nick` was present.

Thank you for changing it.